pgcontents/pgmanager.py

```diff
@@ -73,7 +73,11 @@ def _checkpoints_class_default(self):
         return PostgresCheckpoints
 
     def _checkpoints_kwargs_default(self):
-        kw = super(PostgresContentsManager, self)._checkpoints_kwargs_default()
+        try:
+            klass = PostgresContentsManager
+            kw = super(klass, self)._checkpoints_kwargs_default()
+        except AttributeError:
+            kw = {'parent': self, 'log': self.log}
```

Comment:
This was renamed to `_default_checkpoints_kwargs` in notebook 5, so we can probably just use that instead of hard-coding a set of defaults.